# Single high cost path
geometry_msgs/Point[] path_points  # Points along this trajectory
float32 path_cost                  # Cost value for this path